### ISSUE TITLE
TypeError: Cannot read property 'get' of undefined

### ISSUE THREAD

**Issue by trawlbens** [netlify/netlify-cms#5193]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Applicable Versions:**
 - Netlify CMS version: `netlify-cms@2.10.64`
 - Git provider: `github`
 - Browser version: `Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36`

**CMS configuration**
```
publish_mode: simple
slug:
  encoding: unicode
  clean_accents: false
  sanitize_replacement: "-"
backend:
  name: github
  repo: trawlbens/trawlbens
media_folder: content/posts/images
public_folder: content/posts/images
collections:
  - sortable_fields:
      - commit_date
      - title
      - date
      - commit_author
      - description
    view_filters: []
    canonical_url: false
    meta: {}
    name: posts
    slug: "{{slug}}"
    folder: content/posts
    published: true
    publish: true
    publish_mode: editorial_workflow
    label: Posts
    series: false
    fields:
      - label: Title
        name: title
        widget: string
      - label: Components
        name: components
        widget: list
        field:
          label: Components
          name: components
          widget: relation
          collection: components
          search_fields:
            - name
          value_field: name
          display_fields:
            - "{{name}}"
          default: []
      - label: Date
        name: date
        widget: date
      - label: Writer
        name: writer
        widget: string
      - label: Tags
        name: tags
        widget: list
        field:
          name: tags
          label: Tags
          widget: string
      - label: Keywords
        name: keywords
        widget: list
      - label: Thumbnail
        name: cover_image
        widget: image
      - label: Description
        name: description
        widget: string
      - label: Content
        name: body
        widget: markdown
    create: true
  - sortable_fields:
      - commit_date
      - title
      - commit_author
      - description
    view_filters: []
    canonical_url: false
    meta: {}
    name: updates
    slug: "{{slug}}"
    folder: content/updates
    published: true
    publish: true
    publish_mode: editorial_workflow
    label: Updates
    series: false
    fields:
      - label: Title
        name: title
        widget: string
      - label: Image
        name: image
        widget: image
      - label: YoutubeUrl
        name: youtubeUrl
        widget: string
      - label: Description
        name: description
        widget: markdown
      - label: Tags
        name: tags
        widget: list
        field:
          name: tags
          label: Tags
          widget: string
      - label: CtaButton
        name: ctaButton
        widget: string
    create: true
  - sortable_fields:
      - commit_date
      - name
      - name
    view_filters: []
    canonical_url: false
    meta: {}
    name: components
    slug: "{{slug}}"
    folder: content/components
    published: true
    publish: true
    publish_mode: editorial_workflow
    label: Components
    series: false
    fields:
      - label: Name
        name: name
        widget: string
    create: true

```

**Additional context**

**Comment by erezrokah** [comment]
Hi @trawlbens can you provide a full reproduction for the issue?

**Comment by mattkibbler** [comment]
For @trawlbens and anybody else that sees this error message pop up...

I just had this happening myself for a while, though it didn't seem to affect the functionality of the CMS at all and I wasn't sure what was going on.

I eventually realised it was because I had a demo collection in the CMS that I was playing around with before I created the proper collections that I'm now using. I had an open pull request and active branch for a change on the demo collection, but had then deleted the collection from the CMS config without realising this. 

Even though I was working on other collections. and could no longer see anything to do with the demo collection in the CMS, it seems as though it was still detecting the old demo changes in the background and failing to link them to anything in the CMS which caused the error.
